packaging/install.cmake

```diff
@@ -67,60 +67,67 @@ endif()
 
 if(UNIX)
 
-  # replace MPIEXEC MPIEXEC_NUMPROC_FLAG and MPI_IMPL KHIOPS_MPI_EXTRA_FLAG ADDITIONAL_EN_VAR
+  # Set khiops and khiops_coclustering paths according to the environment (conda, fedora, etc)
+  if(IS_CONDA)
```

Comment:
AFAIU, this would fail for non-standard Conda environments (that is, without `CONDA_PREFIX`), because in this case native install is assumed (see previous comment in this respect).